docs/class1/module2/lab1.rst

```diff
@@ -25,33 +25,33 @@ Lab 2.1: Exploring iApps
 iApp Templates & Deployments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A BIG-IP device has multiple ways to install an iApp onto its platform, including
-TMOS Shell (TMSH), the GUI (TMUI), and the REST Interface. All mechanisms
-are valid and, if needed, can be used in conjunction with each other.
+There are multiple ways to an iApp on BIG-IP. These includes using
```

Comment:
My suggestion: 
 
There are multiple ways to install an iApp on BIG-IP. This includes using